gstatsim.py

```diff
@@ -12,7 +12,6 @@
 from scipy.spatial import distance_matrix
 from scipy.interpolate import Rbf
 from tqdm import tqdm
-import random
```

Comment:
When writing tests, I would suggest to not change the behavior of the package simultaneously.
We can replace the random shuffle in an extra commit